frontend/src/app/components/ExperimentPage.js

```diff
@@ -366,15 +372,27 @@ export default class ExperimentPage extends React.Component {
     });
   }
 
-  feedback() {
-    // Survey link is opened via href link in the template
+  feedback(evt) {
     sendToGA('event', {
       eventCategory: 'ExperimentDetailsPage Interactions',
       eventAction: 'button click',
-      eventLabel: 'Give Feedback'
+      eventLabel: 'Give Feedback',
+      outboundURL: evt.target.getAttribute('href')
+    });
+  }
+
+  showPreFeedbackDialog() {
+    this.setState({
+      showPreFeedbackDialog: true
     });
   }
 
+  handleFeedback(evt) {
+    evt.preventDefault();
+    const { pre_feedback_copy } = this.state.experiment;
+    (pre_feedback_copy === null || pre_feedback_copy === 'undefined') ? this.feedback(evt) : this.showPreFeedbackDialog();
```

Comment:
@chuckharmston added the 'undefined' thing to the test here if the pre_feedback fields are not in the model